--- trunk/plugins/actions/keyframesmanagement/mediadecoder.h	2014/01/26 16:03:30	855
+++ trunk/plugins/actions/keyframesmanagement/mediadecoder.h	2016/10/16 09:36:00	982
@@ -200,7 +200,7 @@
 	{
 		check_missing_plugins();
 		
-		Glib::ustring error = (msg) ? msg->parse().what() : Glib::ustring();
+		Glib::ustring error = (msg) ? Glib::ustring(msg->parse_debug()) : Glib::ustring();
 		
 		dialog_error(_("Media file could not be played.\n"), error);
 		// Critical error, cancel the work.
@@ -214,7 +214,7 @@
 	{
 		check_missing_plugins();
 		
-		Glib::ustring error = (msg) ? msg->parse().what() : Glib::ustring();
+		Glib::ustring error = (msg) ? Glib::ustring(msg->parse_debug()) : Glib::ustring();
 		dialog_error(_("Media file could not be played.\n"), error);
 
 		return true;
--- trunk/plugins/actions/waveformmanagement/mediadecoder.h	2015/06/21 21:15:10	942
+++ trunk/plugins/actions/waveformmanagement/mediadecoder.h	2016/10/16 09:36:00	982
@@ -200,7 +200,7 @@
 	{
 		check_missing_plugins();
 		
-		Glib::ustring error = (msg) ? msg->parse().what() : Glib::ustring();
+		Glib::ustring error = (msg) ? Glib::ustring(msg->parse_debug()) : Glib::ustring();
 		
 		dialog_error(_("Media file could not be played.\n"), error);
 		// Critical error, cancel the work.
@@ -214,7 +214,7 @@
 	{
 		check_missing_plugins();
 		
-		Glib::ustring error = (msg) ? msg->parse().what() : Glib::ustring();
+		Glib::ustring error = (msg) ? Glib::ustring(msg->parse_debug()) : Glib::ustring();
 		dialog_error(_("Media file could not be played.\n"), error);
 
 		return true;
