--- subtitleeditor-0.52.1/configure.ac.orig	2015-06-25 23:05:40.000000000 +0200
+++ subtitleeditor-0.52.1/configure.ac	2016-02-19 21:25:10.363815003 +0100
@@ -253,7 +253,7 @@
 
 if test "$enable_debug" = "yes"; then
   CXXFLAGS="$CXXFLAGS -DDEBUG -g"
-  LDFLAGS="$LDFLAGS -g -ansi -Wall"
+  LDFLAGS="$LDFLAGS -g -Wall"
 fi
 
 # =========================================================================
@@ -265,13 +265,13 @@
 
 if test "$enable_profile" = "yes"; then
   CXXFLAGS="$CXXFLAGS -pg"
-  LDFLAGS="$LDFLAGS -pg -ansi -Wall"
+  LDFLAGS="$LDFLAGS -pg -Wall"
 fi
 
 # =========================================================================
 # gcc flags
 
-CXXFLAGS="$CXXFLAGS -ansi -Wall -Wextra -Wconversion" #-Werror
+CXXFLAGS="$CXXFLAGS -Wall -Wextra -Wconversion" #-Werror
 
 # =========================================================================
 # make use of ccache
